init

```diff
@@ -18,6 +18,18 @@ rescue_shell() {
 	halt -r
 }
 
+is_file_avaiable(){
+    disktmp=$(mktemp)
+    rm -f $disktmp
+    mkdir -p $disktmp || true
+    timeout 10 mount -t auto "$1" $disktmp &>/dev/null
+    [ -f "$disktmp/$2" ] && [ -b "$1" ]
+    status=$?
+    umount -lf $disktmp 2>/dev/null
+    return $status
+}
+
+
```

Comment:
this algoritm try to mount disk and look if file exists. then umount  (SulinOS uses this method)